lib/click.ts

```diff
@@ -1,8 +1,12 @@
-import { InsightsEvent } from "./_sendEvent";
+export interface InsightsSearchClickEvent {
+  eventName: string;
+  userID: string;
+  timestamp: number;
+  indexName: string;
 
-export interface InsightsSearchClickEvent extends InsightsEvent {
   objectID: string | number;
   position: number;
```

Comment:
We cannot provide multiple `position` & `objectID`?

Reply:
I thought I would do that later but I can add it here